Created a module to detect Duplicate Page Aliases

MikeFoden opened the pull request:
Fixes #2

petrsvihlik commented:
Thanks for the contribution, Mike! :+1:  The code looks good. I found just one little issue related to cultures:
- when culture of an alias is set to "All Cultures" (in DB represented as AliasCulture='') it should collide with other aliases no matter what culture they have set.

When this is fixed we can merge your pull request.

MikeFoden commented:
I have updated this to now include 'All Cultures' as it should have always been. 

I also updated the message I created for when an AliasURLPath is being used as a NodeAliasPath for a Tree Node to show which Node has it as an Alias Path.

petrsvihlik commented:
Works great!
